src/main/resources/reference.conf

```diff
@@ -2,6 +2,26 @@ akka.http.server.idle-timeout = 180 s
 akka.http.server.request-timeout = 60 s
 
 resourceTypes = {
+  admin {
```

Comment:
We decided to go with `admin` here, because from what we could tell, `resource_type_admin` below is about administering to specific resource types. I am not super happy with the role as the same name as the resource. @marctalbott do you have any sense of what the original resource type of `resource_type_admin` is for?

Reply:
Agreed @s-rubenstein .... the duplicate naming is super confusing and I think we need more explicit terminology.